src/components/Dashboard/Form/DocumentForm.js

```diff
@@ -62,22 +64,17 @@ export default class DocumentForm extends Component {
         validate={this.validate}
         onSubmit={this.handleSubmit}
       >
-        {({ handleSubmit, submitting, values }) => (
+        {({ handleSubmit,submitting, values }) => (
           <form className={styles.Form} onSubmit={handleSubmit}>
             <FormHeader page={this.state.page} />
             {activePage}
-            <div className="buttons">
-              {page > 0 && (
-                <button type="button" onClick={this.previous}>
-                  Wróć
-                </button>
-              )}
+            <div className={styles.buttons}>
+              {page > 0 && <Button  onClick={this.previous} label="wróć"/>}
               {isLastPage ? (
-                <button type="submit" disabled={submitting}>
-                  Utwórz dokument
-                </button>
+                  <Button  type="submit" disabled={submitting} label="utwórz dokument"/>
+
               ) : (
-                <button type="submit">Dalej</button>
+                  <Button label="dalej"  onClick={(values) =>this.next(values)}/>
```

Comment:
Please remove additional space between Button and onClick and add a space before closing component />

Reply:
Actually this line causes a crash, you are passing to the `next` method event, not values. If you do `console.log(values) you will see that these values are just click event. 
When you look at a previous version, you have there button without passed method, because `next` is handled by submit.
We have checked that and it works fine after small fixes.

Reply:
done